### PR TITLE
audit: bypass artifact check for vifm-osx-0.11

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?
- [ ] Have you successfully run `brew man` locally and committed any changes?

-----

```
==> FAILED
Formula/vifm.rb:4:3: C: https://github.com/vifm/vifm/releases/download/v0.11/vifm-osx-0.11.tar.bz2 looks like a binary package, not a source archive; homebrew/core is source-only.
  url "https://github.com/vifm/vifm/releases/download/v0.11/vifm-osx-0.11.tar.bz2"
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

relates to https://github.com/Homebrew/homebrew-core/pull/63061

### CODE REVIEW

File: Library/Homebrew/rubocops/urls.rb [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Urls < FormulaCop
           https://osxbook.com/book/bonus/chapter8/core/download/gcore
           https://naif.jpl.nasa.gov/pub/naif/toolkit/C/MacIntel_OSX_AppleC_64bit/packages/
           https://artifacts.videolan.org/x264/release-macos/
+          https://github.com/vifm/vifm/releases/download
```
Comment (MikeMcQuaid) [inline Library/Homebrew/rubocops/urls.rb:25]:
```suggestion
          https://github.com/vifm/vifm/releases/download/v0.11/vifm-osx-0.11.tar.bz2
```

as this seems specific to a single release.

Comment (chenrui333) [reply Library/Homebrew/rubocops/urls.rb:25]: yeah, that is right, only for this release.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks @chenrui333!

**Comment by chenrui333** [comment]
Thanks Mike!

**Pull request merged**